README.md

```diff
@@ -74,40 +94,56 @@ or run the following command for Windows:
 $ ./gradlew.bat buildBinaries
 ```
 
-Finally, you can find the binary `wsk` or `wsk.exe` in the bin folder under the OpenWhisk CLI home directory. In
-addition, it is also available under the folder bin/\<os\>/\<cpu arc\>/. For example, if your local operating system is Mac,
-and the CPU architecture is amd64, the binary and its compressed package can also be found under bin/mac/amd64/.
+After the build, you can find the binary `wsk` or `wsk.exe` in the bin folder
+under the OpenWhisk CLI home directory. In addition, it is also available under
+the folder `bin/<os>/<architecture>/`. For example, if your local operating
+system is Mac, and the CPU architecture is amd64, the binary and its compressed
+package can also be found under `bin/mac/amd64/`.
 
-OpenWhisk CLI(`wsk`) is produced in a Docker container during the build process which is copied from the
-Docker container to the local file system in the following directory: bin. This binary will be platform
-specific, it will only run on the operating system, and CPU architecture that matches the build machine.
+OpenWhisk CLI(`wsk`) is produced in a Docker container during the build process
+which is copied from the Docker container to the local file system in the
+following directory: bin. This binary will be platform specific, it will only
+run on the operating system, and CPU architecture that matches the build
 
-If you would like to build the binaries available for all the operating systems and architectures, run the following
-command:
+
+If you would like to build the binaries available for all the operating systems
+and architectures, run the following command:
 
 ```
-$ ./gradlew buildBinaries -PcrossCompileCLI=true
+$ ./gradlew buildBinaries -PcrossCompile=true
 ```
 
-Then, you will find the binaries and their compressed packages generated under the folder bin/\<os\>/\<cpu arc\>/ for each
-operating system and CPU architecture pair. We supports both amd64 and 386 for Linux, Mac and Windows operating systems.
+Then, you will find the binaries and their compressed packages generated under
+the folder `bin/<os>/<cpu arc>/` for each operating system and CPU
+architecture pair. We supports both amd64 and 386 for Linux, Mac and Windows
```

Comment:
`We support` instead of `We supports`. Is the list of architectures correct?

Reply:
Nope.  Re-written.